audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/kafka/KafkaContractTest.java

```diff
@@ -192,8 +192,15 @@ void testGoodAndBadSuccessRequestSet() {
     void testRestEndpointForErrorMessage() {
         // Pass an error message to 'error' rest endpoint
         // When - we POST to the rest endpoint
+        AuditErrorMessage errorMessage = null;
+        try {
+            errorMessage = MAPPER.readValue(ContractTestData.ERROR_REQUEST_JSON, AuditErrorMessage.class);
+        } catch (JsonProcessingException e) {
+            LOGGER.error("Failed to convert error contract test data to objects", e);
+        }
+
         Map<String, List<String>> headers = Collections.singletonMap(Token.HEADER, Collections.singletonList(ContractTestData.REQUEST_TOKEN));
-        HttpEntity<AuditErrorMessage> entity = new HttpEntity<>(ContractTestData.ERROR_REQUEST, new LinkedMultiValueMap<>(headers));
+        HttpEntity<AuditErrorMessage> entity = new HttpEntity<>(errorMessage, new LinkedMultiValueMap<>(headers));
```

Comment:
You've already parsed the object in the ContractTestData class
```suggestion
        HttpEntity<AuditErrorMessage> entity = new HttpEntity<>(ContractTestData.ERROR_REQUEST_OBJ, new LinkedMultiValueMap<>(headers));
```